apps/client/src/routes/protectedRoutes.tsx

```diff
@@ -1,14 +1,16 @@
-import Login from '../pages/Login';
+import LoginForm from '../login/LoginForm';
 import ErrorPage from '../pages/ErrorPage';
 import routes from './routes';
 
+// Temporarily disable protected routes.
+// TO-DO: put routes behind authentication.
```

Comment:
Not blocking: Use `TODO` instead of `TO-DO` so IDEs will recognize the `TODO` token.

Reply:
Not blocking: Put this comment closer to `...routes` to help reader.